### ISSUE TITLE
Enable tcp/443 as fallback for udp 1194

### ISSUE THREAD

**Issue by carmi** [kylemanna/docker-openvpn#27]
Hi @kylemanna,

I was wondering if there is a specific reason you disabled tcp 443 for this build? I know udp is preferable, but I was hoping to find a setup that would fallback to tcp 443 if udp is being blocked by a firewall (or random coffee shop's setup).

**Comment by kylemanna** [comment]
>I was wondering if there is a specific reason you disabled tcp 443 for this
>build? I know udp is preferable, but I was hoping to find a setup that would
>fallback to tcp 443 if udp is being blocked by a firewall (or random coffee
>shop's setup).

The initial incarnation of all of this was intended to be very simple
for the base use case to operate.

That said, in the back of my head I tried to keep it so that power
users could override the config file (i.e. skip the generated file or
modify it) to support whatever they liked.

Support for 443 is definitely available right now if you intend to run
the server dedicated on that port:

    docker run --volumes-from $OVPN_DATA --rm kylemanna/openvpn ovpn_genconfig -u tcp://VPN.SERVERNAME.COM

and later map the port accordingly (note: 443 is the public port 1194 is
inside the container and hard coded to 1194 for simplicity since only
one openvpn daemon runs per container.) 

    docker run --volumes-from $OVPN_DATA -d -p 443:1194/tcp --cap-add=NET_ADMIN kylemanna/openvpn

And things will most likely work as you expect.

What would be an enhancement, would be a method that would allow two
docker-openvpn instances (one on 1194/udp and one on 443/tcp) share the
same EasyRSA PKI certs.

That would require a little bit of work.  Maybe a manually copied config
file that the user would edit and then a script update to allow the
`docker run` command to select the second config file.

Then users would need to manually update their client config files to
support the <connection>... ...</connection> syntax that was removed in
pull request #21
